backend/Tasque.WebAPI/Controllers/AuthController.cs

```diff
@@ -50,5 +52,26 @@ public async Task<IActionResult> Register([FromBody] UserRegisterDto registerInf
             await _service.Register(registerInfo);
             return Ok();
         }
+
+        [HttpPost("restore/request")]
```

Comment:
I think we should make 'restore' a constant and use this constant in the routes.

Reply:
We could move it into separate controller

Reply:
Move what ?